frontends/web/package.json

```diff
@@ -26,7 +26,7 @@
     "flag-icons": "^6.6.6",
     "i18next": "21.6.16",
     "lightweight-charts": "4.1.1",
-    "qr-scanner": "1.4.2",
+    "qr-scanner": "1.3.0",
```

Comment:
Add a comment here to be careful with upgrading this one? Otherwise a regression will come easily.

Also, which Android versions don't work, and how did you figure it out?

Reply:
I don't know how to add comments in JSON 🙈 

For now this is only a draft for testing.

I'll update the commit message.